.appveyor/build_libedit.bat

```diff
@@ -0,0 +1,59 @@
+@echo on
+setlocal
+
+call "%~dp0set_paths.bat"
+
+cd "%BUILDDIR%"
+
+if exist "%BINARIESDIR%\libedit" (
```

Comment:
On CI there are no chances that any of this directory is exists. I have added this check for specifically developers, because I don't see a point of building all PBS deps from start if you have already built.
However we should not error out if directory exists, it should be exit 0, I will change code.